lib/pathspec.rb

```diff
@@ -75,7 +75,7 @@ def drive_letter_to_path(path)
 
   # Generate specs from a filename, such as a .gitignore
   def self.from_filename(filename, type=:git)
-    self.from_lines(File.open(filename, 'r'))
+    self.from_lines(File.open(filename, 'r'), type)
```

Comment:
Well, that's a bit embarrassing. That's what I get for not having a test case there.